tests/end2end/features/downloads.feature

```diff
@@ -30,17 +30,16 @@ Feature: Downloading things from a website.
         And I open data/downloads/issue1243.html
         And I run :hint links download
         And I run :follow-hint a
-        And I wait for "Asking question <qutebrowser.utils.usertypes.Question default='qutebrowser-download' mode=<PromptMode.text: 2> text='Save file to:'>, *" in the log
-        And I run :leave-mode
-        Then no crash should happen
+        And I wait for "Asking question <qutebrowser.utils.usertypes.Question default='qutebrowser-download' mode=<PromptMode.download: 5> text='Save file to:'>, *" in the log
+        Then the error "Download error: No handler found for qute://!" should be shown
```

Comment:
I don't really understand why this changed - is it just better matching based on the log?